### PR TITLE
Correct cluster-api project path

### PR DESCRIPTION
**Pull request by Sunnatillo**
This PR corrects cluster-api project path

### CODE REVIEW

File: scripts/release-crawler/go.mod [hunk 1]
```diff
@@ -6,8 +6,4 @@ require github.com/gomarkdown/markdown v0.0.0-20231115200524-a660076da3fd
 
 require github.com/google/go-github/v57 v57.0.0
 
-require (
```
Comment (jberkus) [inline scripts/release-crawler/go.mod:9]: Can you explain why you're changing the requirements here?

Comment (Sunnatillo) [reply scripts/release-crawler/go.mod:9]: I just run go mod tidy

Comment (Sunnatillo) [reply scripts/release-crawler/go.mod]: Sorry go version were different for me. I removed the changes

**Comment by Sunnatillo** [comment]
@lwkd-maintainers 
PTAL

**Comment by jberkus** [comment]
Yeah, sorry, I keep meaning to merge this when we're not in the middle of editing a weekly edition, and I keep forgetting.

File: scripts/release-crawler/README.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ The collected release notes are stored as html and as MD format to be an easy us
 |kubernetes|kops           |[kops](https://github.com/kubernetes/kops)|
 |kubernetes|kubeadm           |[kubeADM](https://github.com/kubernetes/kubeadm)|
 |kubernetes|ngninx-ingress           |[NGINX Ingress](https://github.com/kubernetes/ngninx-ingress )|
-|kubernetes|cluster-api           |[ClusterAPI](https://github.com/kubernetes/cluster-api)|
+|kubernetes|cluster-api           |[ClusterAPI](https://github.com/kubernetes-sigs/cluster-api)|
```
Comment (chrischdi) [inline scripts/release-crawler/README.md:42]:
```suggestion
|kubernetes|cluster-api           |[Cluster API](https://github.com/kubernetes-sigs/cluster-api)|
```

**Review by chrischdi — COMMENTED** [review]
Let's also fix this naming nit here :-)

**Pull request merged**